categories.tex

```diff
@@ -1231,7 +1231,7 @@ \section{The structure identity principle}
     \indexsee{PH-structure@$(P,H)$-structure}{structure}%
     \indexdef{structure!PH@$(P,H)$-}%
     on $x$.
-  \item For $x,y:X_0$ and $\alpha:Px$, $\;\beta:Py$, to each $f:\hom_X(x,y)$ a mere proposition
+  \item For $x,y:X_0$ and $\alpha:Px$, $\;\beta:Py$, we associate to each $f:\hom_X(x,y)$ a mere proposition
```

Comment:
I'm not sure about this wording.  The current version is ungrammatical, but "we associate" isn't normally something I would say when specifying the data in a definition.  What about just "For $x,y:X_0$, $f:\hom_X(x,y)$ and $\alpha:Px$, $\beta:Py$, a mere propositions..."?

Reply:
Sure, that was the alternative I was considering.  I'll push an update.  (The "we ..." form was meant to be similar to the "we have" in the later \item's.)